src/data/scripts/add-context-by-property-type.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+import json
+
+path_to_property_type_json = "./dist/property-types.json"
+path_to_buildings_csv = "./dist/building-benchmarks.csv"
+
+# Columns we want to rank for and append ranks to each building's data
+building_cols_to_rank = [
+    'GHGIntensity',
+    'TotalGHGEmissions',
+    'ElectricityUse',
+    'NaturalGasUse',
+    'GrossFloorArea',
+    'SourceEUI',
+    'SiteEUI',
+]
+
+#raw building data
+building_data = pd.read_csv(path_to_buildings_csv)
+
+#sorted data based on each property type: the order is alphabetical
+sorted_by_property_type = building_data.groupby("PrimaryPropertyType")
+
+
+def calculateBuildingStatistics():
+    building_staistics_by_property_type = {}
+    all_property_types = pd.read_json("./dist/property-types.json")
+
+    #looping through both all the property types and all teh columns we want to get data on
```

Comment:
Typo, "teh" -> "the"

I also prefer comments with a starting space personally, like `# looping through`. I just think that's more readable but not a blocker 🤷🏻‍♂️